config/routes.rb

```diff
@@ -357,7 +357,6 @@
     get "/async_info/shell_version", controller: "async_info#shell_version", defaults: { format: :json }
 
     get "/future", to: redirect("devteam/the-future-of-dev-160n")
```

Comment:
We'll want to tackle /future in a subsequent PR as well. I left it for now because I'd like to get this out first

Reply:
Can we add a todo note here so that at the very least people don't add more references since we know we want to remove this route?

Reply:
@vaidehijoshi I don't think we need to as I'll be sending this out in a subsequent PR today, just waiting for confirmation that this is the correct way to go about it.